src/pluginplay/detail_/module_manager_pimpl.hpp

```diff
@@ -221,6 +221,13 @@ struct ModuleManagerPIMPL {
     void set_runtime(runtime_ptr runtime) noexcept { m_runtime_ = runtime; }
 
     runtime_type& get_runtime() const { return *m_runtime_.get(); }
+
+    std::vector<type::key> keys() const noexcept {
```

Comment:
Needs to use the typedef alluded to in the previous comment for consistency. This is not no-throw guarantee. Line 227 and 228 can both throw `std::bad_alloc`.

Reply:
Thanks. Done with [63c6de0](https://github.com/NWChemEx-Project/PluginPlay/pull/326/commits/63c6de0a9bb6265b2143d3befaa7487140da8b3b).